modules/@apostrophecms/schema/ui/apos/logic/AposInputRelationship.js

```diff
@@ -274,29 +274,19 @@ export default {
 
       const result = await apos.modal.execute(editor, {
         schema: this.field.schema,
-        item,
-        title: item.title,
-        'model-value': item._fields
+        item
       });
 
-      if (result) {
+      if (result?.image) {
```

Comment:
Is `AposInputRelationship` only for images?

Reply:
Fixed